k8s/configmap.yaml

```diff
@@ -15,79 +15,3 @@ data:
   COMMUNITY: "false"  # Enable this if you want your server to show up in the community servers tab, USE WITH SERVER_PASSWORD!
   SERVER_NAME: "World of Pals"
   SERVER_DESCRIPTION: ""
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: palworld-settings
-data:
-  PalWorldSettings.ini: >-
-    [/Script/OnlineSubsystemUtils.IpNetDriver]
-    NetServerMaxTickRate=60
```

Comment:
IMO this didn't work in the first place. We should make a PR and put this in Engine.ini.